HOW_WE_USE_GITHUB.md

```diff
@@ -91,83 +95,110 @@ At the most basic "bird's eye view" level, sorted issues will fall into the cate
 
 At its core, sorting enables new issues to be placed into these four categories, which helps to ensure that they will be processed at a velocity similar to or exceeding the rate at which new issues are coming in. One of the benefits of actively sorting issues is to avoid engineer burnout and to make necessary work sustainable; this is done by eliminating a never-ending backlog that has not been reviewed by any maintainers.
 
-There will always be broad-scope design and architecture implementations that the `conda` maintainers will be interested in pursuing; by actively organizing issues, the sorting engineers will be able to more easily track and tackle both specific and big-picture goals.
-
-#### Who does the sorting?
-
-Sorting engineers are a `conda` governance [sub-team][sub-team]; they are a group of Anaconda and community members who are responsible for making decisions regarding closing issues and setting feature work priorities, amongst other sorting-related tasks.
+There will always be broad-scope design and architecture implementations that the maintainers will be interested in pursuing; by actively organizing issues, the sorting engineers will be able to more easily track and tackle both specific and big-picture goals.
 
+### Who does the sorting?
 
-#### How do items show up for sorting?
+Sorting engineers are a conda governance [sub-team][sub-team]; they are a group of Anaconda, Inc. and community members who are responsible for making decisions regarding closing issues and setting feature work priorities, among other sorting-related tasks.
```

Comment:
AFAIK the steering council has not voted on such a subteam, so this doesn't exist at the moment.

Also, we shouldn't highlight Anaconda Inc here per se, since the subteam is a community governance group, not related to Anaconda, Inc. Employees may be members of it, but that's coincidental.

```suggestion
Sorting engineers are a conda governance [sub-team][sub-team]; they are a group of community members who are responsible for making decisions regarding closing issues and setting feature work priorities, among other sorting-related tasks.
```

Reply:
An example proposal for a subteam is here: https://github.com/conda-incubator/governance/issues/81

Reply:
yeah, this text was originally written with the expectation that the subteam would be created "soon", I agree this should be rephrased until then